src/openseadragon.js

```diff
@@ -685,19 +685,10 @@
   * @param {OpenSeadragon.Options} options - Viewer options.
   * @returns {OpenSeadragon.Viewer}
   */
-window.OpenSeadragon = window.OpenSeadragon || function( options ){
-
+function OpenSeadragon( options ){
```

Comment:
I don't know enough about Require to know if that's the case. It'd be great to get this nice and solid. On the other hand, I suppose it's not a big deal until it's causing problems for someone.